src/display/editor/editor.js

```diff
@@ -522,6 +534,13 @@ class AnnotationEditor {
     if (!this.#keepAspectRatio) {
       this.div.style.height = `${((100 * height) / parentHeight).toFixed(2)}%`;
     }
+    if (this.#altTextButton) {
+      if (width < 128 || height < 128) {
+        this.#altTextButton.classList.add("small");
+      } else {
+        this.#altTextButton.classList.remove("small");
+      }
```

Comment:
Should we perhaps make the hard-coded value a suitably named constant instead?

In any case, I do believe that this can be simplified a bit:
```suggestion
      this.#altTextButton.classList.toggle("small", width < 128 || height < 128);
```